src/CountryScoresCard.js

```diff
@@ -1 +1,25 @@
+import React from "react";
+
+const CountryScoresCard = (props) => {
+  return (
+    <div className="country-score-card">
+      <p>HIGH SCORES: {props.country.name} </p>
+      <div className="card-info">
+        {props.country.scores
+          .sort((a, b) => (a.s < b.s ? 1 : -1))
+          .map((person) => (
+            <div>
```

Comment:
can you think of any other smaller components that could be used in this component?